integration_tests/features/CoinbaseReorg.feature

```diff
@@ -0,0 +1,78 @@
+@coinbase_reorg
+Feature: Wallet Routing Mechanism
```

Comment:
The Feature name should match the file name. I would suggest WalletMonitoring as a file and feature name

Reply:
Ooops... copy paste error. Will update.